docs/development_suite/api-console/api-design/endpoints.md

```diff
@@ -78,7 +78,14 @@ For the CRUD endpoint it's not possible to set an internal Rewrite. The Internal
 
```

Comment:
With the introduction of listeners management, in this documentation page, we have also to change the image and the information about endpoint creation. See the following paragraph https://docs.mia-platform.eu/docs/development_suite/api-console/api-design/endpoints#details

Moreover, even info about required fields in endpoint creation has to be updated by adding listeners field info

Reply:
updated screenshots